net/curl/Portfile

```diff
@@ -192,10 +192,23 @@ if {${name} eq ${subport}} {
                                 path:share/curl/curl-ca-bundle.crt:curl-ca-bundle
         configure.args-replace  --without-ssl --with-ssl=${prefix}
         configure.args-append   --with-ca-bundle=${prefix}/share/curl/curl-ca-bundle.crt
+
+        # remove after 2021-07-01
+        if {${os.platform} eq "darwin"} {
+            notes "
+Please note that the default SSL backend was recently changed from OpenSSL to \
+the native Secure Transport backend. To reenable binary packages, run:
+  $ ${0} upgrade --enforce-variants curl -ssl +darwinssl
```

Comment:
This doesn't work: `can't read "0": no such variable`. In other ports, we just write it out as `sudo port`. I don't think there's a better way right now.